src/public/packages/backpack/crud/js/selects.js

```diff
@@ -0,0 +1,223 @@
+let refreshOptionList = function (element, $field, $refreshUrl) {
+    return new Promise(function (resolve, reject) {
+        $.ajax({
+            url: $refreshUrl,
+            data: {
+                'field': $field
+            },
+            type: 'GET',
+            // async: false,
+            success: function (result) {
+                //console.log(result);
+                $(element).attr('data-options-for-select', JSON.stringify(result));
+                resolve(result);
+            },
+            error: function (result) {
+                // Show an alert with the result
+
+                reject(result);
+            }
+        });
+    });
+};
+
+
+function fillSelectOptions(element, $created = false, $multiple = false) {
```

Comment:
This fills the selects with the options provided on page load or after entity creation.